src/Components/__tests__/CountdownTimer.jest.tsx

```diff
@@ -104,7 +104,7 @@ describe("CountdownTimer", () => {
 
     const text = timer.text()
     expect(text).toMatchInlineSnapshot(
-      `"Time remaining01d 00h 00m 00s leftRespond by Dec 4, 8:50am ESTExpired offers end the negotiation process permanently."`
+      `"01d 00h 30m 00s leftRespond by Dec 4, 8:50am ESTExpired offers end the negotiation process permanently."`
```

Comment:
This spec was just wrong. As you can see it wasn't asserting on that half hour difference. It works now so 🤷